versioned_docs/version-24.04/upgrade/upgrade-from-23-10.md

```diff
@@ -91,27 +91,26 @@ apt update
 
 ### Upgrade the Centreon solution
 
-> Make sure all users are logged out from the Centreon web interface
-> before starting the upgrade procedure.
+1. Make sure all users are logged out from the Centreon web interface before starting the upgrade procedure.
 
-If you have installed Business extensions, update the Business repository to version 24.04.
+2. If you have installed Business extensions, update the Business repository to version 24.04.
```

Comment:
2. If you have installed Business extensions, delete the configuration of repository 23.10 : 

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```shell
rm /etc/yum.repos.d/centreon-business-23.10.repo
```

</TabItem>

<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```shell
rm /etc/yum.repos.d/centreon-business-23.10.repo
```

</TabItem>

<TabItem value="Debian 11" label="Debian 11">

```shell
rm /etc/apt/sources.list.d/centreon-business.list
```

</TabItem>
</Tabs>